security/pkg/nodeagent/cache/secretcache.go

```diff
@@ -866,6 +866,7 @@ func (sc *SecretCache) generateSecret(ctx context.Context, token string, connKey
 	}
 	options := pkiutil.CertOptions{
 		Host:       csrHostName,
+		Org:        "Istio",
```

Comment:
what is the impact of this on existing users of Citadel?

Reply:
And why ? 

If we want to customize Org - should be an option. 

Org is used in K8S for example with specific meaning ( I think it's used as a group ), and we'll sign with k8s CA as well. I don't want to have unexpected side-effects.

Reply:
noted, 
Thanks @costinm, @howardjohn